### DISCUSSION TITLE
problem of config.yaml

### DISCUSSION THREAD

**Discussion by github-aaliu** [STVIR/pysot#185] [General] [answered]
感觉代码中自带的config.yaml 的配置有点问题，而且和作者提供的自己训练的模型中附带的config.yaml也不一样

**Comment by github-aaliu** [comment]
感谢你的回复，我训的是siamrpn_alex_dwxcorr，总的参数列表comfig.py里有个参数__C.BACKBONE.TRAIN_EPOCH = 10，意思是backbone从第10个epoch才开始训练，这应该是针对backbone有预训练模型的情况，而alexnet没有预训练模型（或者也没加载），但是他本身的config.yaml也没去覆盖这个参数，其中还有很多的参数问题，我只是举了一个

**Reply by ZhiyuanChen** [reply]
是存在一些hard code的地方，需要手动修改

**Comment by chun-chun123** [comment, marked as answer]
训练时候应该选择多gpu文件夹下的config.yaml

**Comment by Dtappledoghuati** [comment]
请问有成功跑通过siammask吗，我训练总是出问题，应该是config.yaml文件的参数配置问题

**Reply by 422261883** [reply]
> 请问有成功跑通过siammask吗，我训练总是出问题，应该是config.yaml文件的参数配置问题

请问您跑通了siammask吗，我看作者回复了 #TODO那个，说还未完成，是跑不了siammask吗

**Comment by Dtappledoghuati** [comment]
您好，我已经放弃了siammask，没有跑通



在 2020-12-05 17:12:05，"422261883" <notifications@github.com> 写道：

请问有成功跑通过siammask吗，我训练总是出问题，应该是config.yaml文件的参数配置问题

请问您跑通了siammask吗，我看作者回复了 #TODO那个，说还未完成，是跑不了siammask吗
